Tools/Schema.NET.Tool/GeneratorModels/GeneratorSchemaObject.cs

```diff
@@ -2,8 +2,14 @@ namespace Schema.NET.Tool.GeneratorModels
 {
     public class GeneratorSchemaObject
     {
-        public string? Layer { get; set; }
+        public GeneratorSchemaObject(string name, string layer)
```

Comment:
I think it would make more logical sense for the layer parameter to come first since it is higher up in the hierarchy.